README.rst

```diff
@@ -57,30 +48,31 @@ To build the documentation on your checkout, simply run::
 
   make docs
 
-We plan to get this published in the near future, and this README will be
-updated when that happens.
-
 Contributions
 *************
 
 All new code should include tests that excercise the code and prove that it
 works, or fixes the bug you are trying to fix.  Any Pull Request without tests
-will not be accepted.
+will not be accepted. See `CONTRIBUTING <CONTRIBUTING.rst>`_ for more details.
```

Comment:
I notice this works here in github, but not on the sphinx-generated doc.  I can update it to be just text, if that is what everyone prefers, just let me know.

Reply:
Interesting! Personally, I think CONTRIBUTING.rst without link might be just as well in this case. Easier for RTD readers to understand they need to look for a file.

Reply:
K, updating